cdap-app-fabric/src/main/java/io/cdap/cdap/internal/provision/ProvisioningService.java

```diff
@@ -224,7 +229,7 @@ public ClusterStatus getClusterStatus(ProgramRunId programRunId, ProgramOptions
             Networks.getAddress(cConf, Constants.NETWORK_PROXY_ADDRESS), null, null);
       }
       context = createContext(cConf, programOptions, programRunId, userId, properties,
-          defaultSSHContext);
+          defaultSSHContext, null);
```

Comment:
Why not OTHERS? It's better to have less nullable things. Also in other cases we use OTHERS as default

Reply:
done.